symengine/parser/tokenizer.cpp

```diff
@@ -0,0 +1,380 @@
+/* Generated by re2c 1.1.1 */
```

Comment:
I ran the parser and formatted and got the exact same file now with re2c 1.1.1.

Reply:
Thanks.